macOS/PostEditor/PostTextEditingView.swift

```diff
@@ -2,65 +2,98 @@ import SwiftUI
 
 struct PostTextEditingView: View {
     @ObservedObject var post: WFAPost
-    @Binding var updatingTitleFromServer: Bool
-    @Binding var updatingBodyFromServer: Bool
-    @State private var isHovering: Bool = false
+    @Binding var updatingFromServer: Bool
     @State private var appearance: PostAppearance = .serif
-    private let bodyLineSpacingMultiplier: CGFloat = 0.5
+    @State private var combinedText = ""
 
     var body: some View {
-        VStack {
-            TextField("Title (optional)", text: $post.title)
-                .textFieldStyle(PlainTextFieldStyle())
-                .padding(.horizontal, 4)
-                .font(.custom(appearance.rawValue, size: 26, relativeTo: .largeTitle))
-                .onChange(of: post.title) { _ in
-                    if post.status == PostStatus.published.rawValue && !updatingTitleFromServer {
-                        post.status = PostStatus.edited.rawValue
-                    }
-                    if updatingTitleFromServer {
-                        updatingTitleFromServer = false
-                    }
-                }
-                .padding(4)
-                .background(Color(NSColor.controlBackgroundColor))
-                .padding(.bottom)
-            ZStack(alignment: .topLeading) {
-                if post.body.count == 0 {
-                    Text("Write…")
-                        .foregroundColor(Color(NSColor.placeholderTextColor))
-                        .padding(.horizontal, 4)
-                        .padding(.vertical, 2)
-                        .font(.custom(appearance.rawValue, size: 17, relativeTo: .body))
-                }
-                TextEditor(text: $post.body)
+        ZStack(alignment: .topLeading) {
+            if combinedText.count == 0 {
+                Text("Write…")
+                    .foregroundColor(Color(NSColor.placeholderTextColor))
+                    .padding(.horizontal, 5)
                     .font(.custom(appearance.rawValue, size: 17, relativeTo: .body))
-                    .lineSpacing(17 * bodyLineSpacingMultiplier)
```

Comment:
I'm unable to test, so I'm not sure -- but does this mean the Mac app won't have the same line spacing as the iOS app?

Reply:
Ah yeah, forgot to add that back! Let me fix that.

Reply:
Before 2e56d62:

<img width="1288" alt="Screen Shot 2020-11-24 at 14 31 53" src="https://user-images.githubusercontent.com/387655/100142940-90721380-2e62-11eb-92dd-775afe39b74a.png">

After 2e56d62:

<img width="1288" alt="Screen Shot 2020-11-24 at 14 34 09" src="https://user-images.githubusercontent.com/387655/100142969-99fb7b80-2e62-11eb-9037-8c7063c3b724.png">

Reply:
Looks great!